src/lib/features/dependent-features/dependent-features-read-model-type.ts

```diff
@@ -1,5 +1,7 @@
+import { IDependency } from '../../types';
+
 export interface IDependentFeaturesReadModel {
     getChildren(parent: string): Promise<string[]>;
-    getParents(child: string): Promise<string[]>;
+    getParents(child: string): Promise<IDependency[]>;
```

Comment:
currently we only need string but I want to have API that will be prepared for the upcoming enhancements (variants, enabled/disabled)